tests/ltp/README.md

```diff
@@ -439,7 +439,7 @@ FAILING
 | /ltp/testcases/kernel/syscalls/getitimer/getitimer02 | 0 | 0 | 0 |
 | /ltp/testcases/kernel/syscalls/getitimer/getitimer03 | 1 | 1 | 1 |
 | /ltp/testcases/kernel/syscalls/getpagesize/getpagesize01 | 1 | 1 | 1 |
-| /ltp/testcases/kernel/syscalls/getpeername/getpeername01 | 1 | 1 | 1 |
+| /ltp/testcases/kernel/syscalls/getpeername/getpeername01 | 0 | 0 | 0 |
```

Comment:
Did getpeername01 test pass in the past?

Reply:
Same as bind03, there was another PR to remove it